src/amoc_test_event.erl

```diff
@@ -0,0 +1,55 @@
+-module(amoc_test_event).
+
+-behaviour(gen_event).
+
+-export([init/1, handle_event/2, handle_call/2, handle_info/2, terminate/2, code_change/3]).
```

Comment:
Line 5 is too long: -export([init/1, handle_event/2, handle_call/2, handle_info/2, terminate/2, code_change/3])..